kernel/ksu.c

```diff
@@ -89,7 +89,7 @@ static uid_t __manager_uid;
 
 static bool is_manager()
 {
-	return __manager_uid == current_uid().val;
+	return __manager_uid == current_uid().val || 0 == current_uid().val;
```

Comment:
we should not allow any root process to be manager, if you want to get verion, just allow root process to get version, avoiding widen the scope.

Reply:
And also, you should put this to another PR, don't mess it up.

Reply:
If don't allow this, we can:
Allow non-manager to excute `CMD_GET_VERSION`;
Hardcode it
which is better？

Reply:
I think this is not a breaking change, so I didn't put it into another PR.

Reply:
Each time you push commits to this branch, the actions will compile the whole kernel for 9 times.
I've canceled action runs for many times.

Reply:
> Each time you push commits to this branch, the actions will compile the whole kernel for 9 times. I've canceled action runs for many times.

Sorry for that

Reply:
> I think this is not a breaking change, so I didn't put it into another PR.

This IS a breaking change.

Reply:
You just need to get the version, why allow all root process do manager operation?

Reply:
> You just need to get the version, why allow all root process do manager operation?

My fault, sorry.